webcompat/static/js/lib/labels.js

```diff
@@ -150,15 +150,16 @@ issues.LabelEditorView = Backbone.View.extend({
     }
   },
   updateView: function(evt) {
-    // We try to make sure only one "status"-type label is set
-    // If the change event comes from a "status"-type label,
-    // enumerate all checked "status"-type labels and uncheck
+    // We try to make sure only one "status" or "priority"-type label is set
+    // If the change event comes from a "status" or "priority"-type label,
+    // enumerate all checked "status" or "priority"-type labels and uncheck
     // the others.
     var checked;
     if (
-      $(evt.target).data("remotename").match(/^status/) && evt.target.checked
+      $(evt.target).data("remotename").match(/^(status|priority)/) && evt.target.checked
```

Comment:
Just a tiny comment. May be we can declare a var for `$(evt.target).data("remote name")` that we use twice.